modules/marker_control/settings.sqf

```diff
@@ -12,7 +12,7 @@
 // Adds all markers that are on the named layer
 
 // Example:
-// [blufor, "markerBlu", true] call FUNC(Add);
-// [[blufor, civilian], "markerBlu2", false] call FUNC(Add);
-// [opfor, "markerRed", true] call FUNC(Add);
+// [blufor, "markerBlu", true] call FUNC(Addlayer);
```

Comment:
Not all calls need to be AddLayer: 

if you want to affect all markers on a layer: addLayer
if you want to affect only a named marker: add